style.css

```diff
@@ -0,0 +1,317 @@
+@import 'https://fonts.googleapis.com/css?family=Lato:400,700';
+
+#body {
+  background-color: #eee;
+  font-family: 'Lato', sans-serif;
+}
+
+#page-wrapper {
+  position: relative;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  color: #000;
+  text-decoration: none;
+}
+
+.container {
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.nav-link {}
```

Comment:
You don't put anything inside this nav-link, so you can delete it for a clean code.

Reply:
- Kindly, phrase the comments politely, as suggestions instead of orders. For example:
        This is a polite suggestion: **"I think it would be better to remove `nav-link`. ....."**
- [Optional] Great job for adding the comment politely and with reasons 👍,  I think it would be better if you use a couple of emojis and/or gifs in the review, they make the code review more human-friendly 👽🤖.